docs/scripts/make-all.rst

```diff
@@ -147,6 +147,10 @@ Configurations
   Build with ``-DCMAKE_BUILD_TYPE=Release``.
   The compiler settings will default to ``-O3 -DNDEBUG``.
 
+If no configuration is specified, the CMake listfile will default to
+``RelWithDebInfo``.
+This will probably be changed to ``Release`` in a future release.
```

Comment:
Dont think we need this that something "will probably be changed to". 

This can be updated when we change the deault

Reply:
Actually, I think we should make a _point_ of telling people in advance that we expect something to change.

Especially because we do such a poor job of telling people _after_ we've made the change.